src/Codeception/Lib/Driver/AmazonSQS.php

```diff
@@ -17,10 +17,14 @@ class AmazonSQS implements Queue
      */
     public function openConnection($config)
     {
-        $this->queue = SqsClient::factory([
-            'credentials' => new Credentials($config['key'], $config['secret']),
+        $params = [
             'region' => $config['region']
-        ]);
+        ];
+        if (!empty($config['profile'])) {
+            $params['profile'] = $config['profile'];
```

Comment:
Please document the new parameter in Config section of Queue documentation https://github.com/Codeception/Codeception/blob/2.1/src/Codeception/Module/Queue.php#L39